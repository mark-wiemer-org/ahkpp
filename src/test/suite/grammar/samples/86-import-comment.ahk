#Include
#Include file.ahk
#Include file.ahk ;comment
#Include C:\My Scripts
#Include C:\My Scripts ;comment
