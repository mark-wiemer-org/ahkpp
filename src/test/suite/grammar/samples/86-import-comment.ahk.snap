>#Include
#^^^^^^^^ source.ahk keyword.control.import.ahk
>#Include file.ahk
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
>#Include file.ahk ;comment
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
#                 ^^^^^^^^^ source.ahk keyword.control.import.ahk comment.line.semicolon.import.ahk
>#Include C:\My Scripts
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
>#Include C:\My Scripts ;comment
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
#                      ^^^^^^^^^ source.ahk keyword.control.import.ahk comment.line.semicolon.import.ahk
>